src/common/css/devdocs.css

```diff
@@ -36,3 +36,10 @@
   font-size: 1.3em;
   margin: 1.5em 0 0.5em 0;
 }
+
+/* Makes all the big code snippets scrollable */
+.content-wrap > div[class*="language-"].highlighter-rouge .highlight {
```

Comment:
This is probably over-specific selector, but it's okay. I will move this functionality to the theme (https://github.com/magento-devdocs/devdocs-theme/) and will make sure we use generic selector.

Reply:
I wasn't aware of this project, however it seems that `.highlighter-rouge` isn't needed here, but wasn't sure about that.